app/js/component/data/github_issues.js

```diff
@@ -426,6 +437,18 @@ function (defineComponent, withAuthTokeFromHash, repositoriesURLs, withPopoverTe
       this.attr.issues = [];
     };
 
+    this.showMessage = function(message){
+      $('#messageModal').modal('toggle');
+      $('#messageModal .modal-title').text(message.title);
+      $('#messageModal .modal-body p').text(message.body);
+    };
+
+    this.showPermissonErrorMessage = function(){
+      this.showMessage({
+        title: 'Error',
```

Comment:
I like the "Oops..." better, instead of saying "Error" in the title and description. Thoughts?